cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/utils/saved_search.js

```diff
@@ -249,6 +249,7 @@ export const generateAllTestConfigurations = () => {
  * @param {QueryEnhancementLanguage} language - query language
  */
 export const setDatePickerDatesAndSearchIfRelevant = (language) => {
+  cy.log(`setDatePickerDatesAndSearchIfRelevant with language: ${language}`);
```

Comment:
Nit: seems this change is unrelated to this PR, is it by intention?

Reply:
Yeah, this was mainly to debug the flaky(?) test that we are seeing on CI10. But i'm seeing that it's not really helping, so I am planning on removing it